elixir/apps/domain/test/support/fixtures/token_fixtures.ex

```diff
@@ -133,12 +126,11 @@ defmodule Domain.TokenFixtures do
   defp infrastructure_token_secrets(attrs) do
     attrs =
       attrs
-      |> Map.put_new_lazy(:secret_nonce, &generate_secret_nonce/0)
       |> Map.put_new_lazy(:secret_fragment, &generate_secret_fragment/0)
       |> Map.put_new_lazy(:secret_salt, &generate_salt/0)
 
     Map.put_new_lazy(attrs, :secret_hash, fn ->
-      compute_secret_hash(attrs.secret_nonce, attrs.secret_fragment, attrs.secret_salt)
+      compute_secret_hash("", attrs.secret_fragment, attrs.secret_salt)
```

Comment:
Not blocking, just another place we're passing in an empty string that may be difficult to understand why at first glance.